src/components/Contacts/ContactDetails/ContactTaskRow/ContactTaskRow.test.tsx

```diff
@@ -0,0 +1,279 @@
+import { MuiThemeProvider } from '@material-ui/core';
+import { render } from '@testing-library/react';
+import React from 'react';
+import {
+  ActivityTypeEnum,
+  ResultEnum,
+} from '../../../../../graphql/types.generated';
+import { gqlMock } from '../../../../../__tests__/util/graphqlMocking';
+import theme from '../../../../theme';
+import { ContactTaskRow } from './ContactTaskRow';
+import {
+  ContactTaskRowFragment,
+  ContactTaskRowFragmentDoc,
+} from './ContactTaskRow.generated';
+
+const startAt = '2021-04-12';
+
+describe('ContactTaskRow', () => {
+  it('should render not complete', async () => {
+    const task = gqlMock<ContactTaskRowFragment>(ContactTaskRowFragmentDoc, {
+      mocks: {
+        startAt,
+        result: ResultEnum.None,
+      },
+    });
+
+    const { queryByText } = render(
+      <MuiThemeProvider theme={theme}>
+        <ContactTaskRow task={task} />
+      </MuiThemeProvider>,
+    );
+
+    expect(queryByText(task.subject)).toBeInTheDocument();
```

Comment:
I this is better because `findBy*` throws errors that show you more info about why it wasn't found instead of just null and `toBeVisible` to make sure it's actually on screen.
```suggestion
    expect(findByText(task.subject)).toBeVisible();
```
But what you have is ok too.